### PR TITLE
add info on how to connect to mongo db cli

### PR DESCRIPTION
**Pull request by peetucket**
## Why was this change made?

Add info on how to connect to the mongo for running queries using the CLI (useful for https://github.com/LD4P/sinopia_api/issues/215)

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -78,6 +78,32 @@ To run a single test (and see console messages):
 
 Or temporarily change the test description from `it("does something")` to `it.only("does something")` and run the single test file with `npx`.
 
+### Connecting to the Mongo CLI
+
+#### Localhost
+
+Note, adjust the docker container name as needed (if for example you are using the sinopia_editor instance of mongo:
```
Comment (mjgiarlo) [inline README.md:85]:
```suggestion
Note, adjust the docker container name as needed (if for example you are using the sinopia_editor instance of mongo):
```

File: README.md [hunk 2]
```diff
@@ -78,6 +78,32 @@ To run a single test (and see console messages):
...
+
+`docker exec -it sinopia_api_mongo_1 mongo mongo`
```
Comment (mjgiarlo) [inline README.md:87]: > yo dawg i herd u like mongo

File: README.md [hunk 3]
```diff
@@ -78,6 +78,32 @@ To run a single test (and see console messages):
...
+
+#### Dev/Stage/Prod
+
+You will first need to ssh to the AWS Bastion and then get to mongo from there.
```
Comment (mjgiarlo) [inline README.md:91]:
```suggestion
You will first need to ssh to the AWS bastion host and then get to mongo from there.
```

File: README.md [hunk 4]
```diff
@@ -78,6 +78,32 @@ To run a single test (and see console messages):
...
+
+1. Get the dev/stage .pem or prod .pem file (ask other devs) and place at `~/.ssh/sinopia-aws-bastion.pem` on your laptop.
```
Comment (mjgiarlo) [inline README.md:93]:
```suggestion
1. Get the dev/stage/prod .pem file (ask other devs) and place at `~/.ssh/sinopia-aws-bastion.pem` on your laptop.
```

**Comment by peetucket** [comment]
Accepted all your suggestions in a single new commit.

**Review by mjgiarlo — APPROVED** [review]

**Pull request merged**